src/app/components/Table/TablePagination.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react'
+import { useSearchParams } from 'react-router-dom'
+import { Pagination } from '../Pagination'
+
+type TablePaginationProps = {
+  count: number
+  rowsNumber: number
+}
+
+export const TablePagination: FC<TablePaginationProps> = ({ count, rowsNumber }) => {
+  const [searchParams, setSearchParams] = useSearchParams()
+  const offsetSearchQuery = searchParams.get('offset')
+  const offset = offsetSearchQuery ? parseInt(offsetSearchQuery, 10) : 0
```

Comment:
This could potentially throw. Would be nice to wrap in try/catch.

Reply:
It can throw when this meet what conditions ?

Reply:
The string could be invalid integer, if this is coming from url.

Reply:
My bad, I meant the bellow code for page calculation.

Reply:
aa Ok I can see it now, thanks